### PR TITLE
Closes #26174: fix delete time range prompt text punctuation

### LINKED ISSUE #26174: Fix puntucation in delete time frame texts
Relates to [this](https://github.com/mozilla-l10n/android-l10n/pull/475)

Fix punctuation for the `delete_all_history_group_prompt_message` quotation.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21107)

### PR DESCRIPTION
**Pull request by mavduevskiy**
Closes #26174 

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #26174

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1142,7 +1142,7 @@
     <string name="deleting_browsing_data_in_progress">Deleting browsing data&#8230;</string>
 
     <!-- Dialog message to the user asking to delete all history items inside the opened group. Parameter will be replaced by a history group name. -->
-    <string name="delete_all_history_group_prompt_message">Delete all sites in \"%s\"</string>
+    <string name="delete_all_history_group_prompt_message">Delete all sites in \“%s\”</string>
```
Comment (flodolo) [inline app/src/main/res/values/strings.xml:1145]: There's no need to escape the typographic quotes

Comment (gabrielluong) [reply app/src/main/res/values/strings.xml:1145]: Good catch, I was also in the middle of pointing this out.

**Review by gabrielluong — APPROVED** [review]
r+ assuming the changes are addressed.

**Comment by gabrielluong** [comment]
> Closes #26174
> 
> ### Pull Request checklist
> 
> ### GitHub Automation
> Used by GitHub Actions.

Just an FYI, the github action will always link the issues for closure automatically. This is to just reduce the manual burden of writing out "Closes #26174". "Closes" and "Fixes" are both keywords that will autoclose the issue. The github action will run every single time on any pull request updates so I wouldn't go out of your way to fight the edits.

**Pull request merged**